lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -175,26 +191,27 @@ define((require, exports, module) ->
       @contextPreviewExists = false
       @popoverHandler.destroy()
 
-    handleCurrentContext: =>
+    handleCurrentContext: => setTimeout((=>
```

Comment:
add a comment about `setTimeout`? I saw that you removed one such call recently, and now you add one. Dunno if it is the same one, but anyway, it is good to have the reason of delayed invocation written explicitly and warning subsequent editors about possible side-effects of removing `setTimeout` :)